config/bridge-deployed.config.js

```diff
@@ -0,0 +1,11 @@
+const baseConfig = require('./base.config')
+
+const id = `${baseConfig.id}-bridge-deployed`
+
+module.exports = {
+  ...baseConfig.bridgeConfigBasic,
+  ...baseConfig.bridgeMapperConfig,
+  event: 'BridgeMappingAdded',
```

Comment:
I noticed that there is no listener to `BridgeMappingRemoved`. I think it would be useful to listen to that event in order to stop listening to a bridge if it was removed from the contracts. (a bridgeMapping could be added with incorrect parameters or any other reason)

Reply:
Yes, we have thought about this event but I wasn't sure what will happen if by some race condition the `BridgeMappingRemoved` will be processed before `BridgeMappingAdded` as they are on different watchers, so one could start earlier...
Do you have any suggestions?

Reply:
Right, race conditions would be hard to avoid since they are different independent watchers.

One idea to avoid this, could be to instead of having two events: `BridgeMappingAdded` and `BridgeMappingRemoved` on the contract, we can have only one event, something like `BridgeMappingUpdated` where one parameter of the event could be `bool status` to indicate if it is active or not. So here we would listen to that event and act accordingly to update the list.

I don't know if it is the best option, but I think it could work. This is totally open to discussion and other options are welcome. Any thoughts on this @akolotov ?

Reply:
@bejavu we should talk about this.

Reply:
Hello @patitonar, we've changed the event to be `BridgeMappingUpdated` as suggested - see https://github.com/poanetwork/token-bridge/pull/131/commits/cd08c1d467d886e1c88b75fccf12f9203e237e15